components/service-operator/apis/access/v1beta1/principal_types.go

```diff
@@ -65,6 +68,8 @@ type Principal struct {
 type PrincipalSpec struct {
 	// Secret name to be used for storing relevant instance secrets for further use.
 	Secret string `json:"secret,omitempty"`
+	// Name of a service account to trust with access to this Principal, if any.
+	TrustServiceAccount string `json:"trustServiceAccount,omitempty"`
```

Comment:
I don't see this field in the CRD in `charts/gsp-cluster/templates/service-operator/crd/access.govsvc.blah` ... did you run `make update-chart` ? .... it's an optional field, so I guess it doesn't make much difference, but it might affect validation